gateway/src/api/auth_layer.rs

```diff
@@ -290,46 +280,6 @@ where
     }
 }
 
-fn extract_token_expiration(token: String) -> Result<Duration, StatusCode> {
```

Comment:
I assume the reasoning here is that since we have such a large buffer in the cache, we don't need to know the exact expiration time of the token? I suppose if the time it takes for gateway to receive the token from auth  is significant we have a bigger problem.

Reply:
Yes, with the bigger buffer it feels like doing calculations becomes more complex and might have weird and unexpected knock on effects. But then again, setting `EXP_MINUTES` to less than 5 min will also have weird knock on effects here :sweat_smile: